### ISSUE TITLE
Some clickable elements are missing clear visual focus indicators

### ISSUE THREAD

**Issue by lesleyjanenorton** [mozilla/blurts-server#2981]
Tab navigation is currently a bit convoluted because some clickable/focus-able elements don't have clear visual focus indicators. Treatments for :hover and :active are also missing in some cases.

Noticed so far:
- Instances of `a.button`
- Instances of `button.primary` 
- `button.close` in dialog
